### PR TITLE
[Issue #4] Bug in the implementation of Api.GetApps() leading to users with id 0 getting ignored.

### PR DESCRIPTION
**Pull request by amaa-99**
Fixed a bug in the implementation of Api.getApps() leading to users with id 0 getting ignored and subsequently to the returned apps list being empty.

### CODE REVIEW

File: app/src/main/java/dev/ukanth/ufirewall/Api.java [hunk 1]
```diff
@@ -1422,7 +1422,7 @@ public static List<PackageInfoData> getApps(Context ctx, GetAppList appList) {
                 Matcher m = p.matcher(user.toString());
                 if (m.find() && m.groupCount() > 0) {
                     int id = Integer.parseInt(m.group(1));
-                    if (id > 0) {
+                    if (id >= 0) {
```
Comment (ukanth) [inline app/src/main/java/dev/ukanth/ufirewall/Api.java:1425]: uid 0 is root . That's handled separately. We don't need to add it here.

Comment (amaa-99) [reply app/src/main/java/dev/ukanth/ufirewall/Api.java:1425]: I got the issue that at least in one of my (rooted) dev devices (I believe the one running LineageOS 18.1 (i.e. Android 11)) the only user reported at that point is the one with id 0...
If the user with id 0 is intended to be handled elsewhere, then probably the problem lies there.
Could you point me in that direction?

Comment (amaa-99) [reply app/src/main/java/dev/ukanth/ufirewall/Api.java:1425]: p.s. I've dropped the other commits (there are individual pulls for each of those).

Comment (ukanth) [reply app/src/main/java/dev/ukanth/ufirewall/Api.java:1425]: https://github.com/ukanth/afwall/blob/59d20c083cae02608a3dcf0c754af76921bdf373/app/src/main/java/dev/ukanth/ufirewall/Api.java#L257C6-L265C7 - specialAndroidAccounts

Comment (amaa-99) [reply app/src/main/java/dev/ukanth/ufirewall/Api.java:1425]: On the affected devices the listOfUids will always be empty, and therefore the packagesForUser list will empty too:
```
            if(G.supportDual()) {
                packagesForUser  = getPackagesForUser(listOfUids);
            }
```

**Pull request closed without merging**